tests/playwright/sanity/includes/widgets/google-maps.test.ts

```diff
@@ -15,7 +15,11 @@ test( 'Verify Google maps controls', async ( { page, apiRequests }, testInfo ) =
 	await wpAdmin.openNewPage();
 	await editor.closeNavigatorIfOpen();
 	await editor.addWidget( 'google_maps' );
-	await googleMapsWidget.setGoogleMapsParams( { location, zoom, height } );
```

Comment:
This seems better than your alternative.
Maybe you could change the function signature to `setGoogleMapsParams(location, zoom, height)` instead of `setGoogleMapsParams({})`

Reply:
@yotamselementor Read the previous reply.

Reply:
I still think my comment stands

Reply:
Reverted.